packages/rancher-logging/overlay/app-readme.md

```diff
@@ -1,3 +1,8 @@
-Rancher Logging, powered by the [Banzai Logging Operator](https://banzaicloud.com/docs/one-eye/logging-operator/), allows users to configure complex logging pipelines with a few simple resources.
-To collect logs from a single namespace, users can create an [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) to define where logs are stored and a corresponding [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) to describe what logs to include and what outputs to send to.
-To collect logs from the entire cluster, users can similarly create a [ClusterOutput](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) and a corresponding [ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/). 
+# Rancher Logging
+
+Rancher Logging, powered by the [Banzai Logging Operator](https://banzaicloud.com/docs/one-eye/logging-operator/). It allows users to configure complex logging pipelines with a few simple resources:
+
+- [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) configures where to store the collected logs for the namespace it is created in
+- [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) configures what logs in the namespace are sent to the configured outputs. 
+- [ClusterOutput](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) configures where to store logs collected for the cluster.
+- [ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/) configures what logs to direct to the150150 cluster outputs.
```

Comment:
Is this intentional? ```the150150```

Reply:
That is a puppy walking across the keyboard I'll remove it.